test/qa/lib/dns.py

```diff
@@ -1,45 +1,47 @@
-from lib import logging
-import sh
+import dns.resolver
 
-# returns True when NordVPN DNS servers are used
-def _is_set_to_nord():
-    return "103.86.96.100" and "103.86.99.100" in sh.cat("/etc/resolv.conf")
 
+# Used for test parametrization.
+DNS_NORD = ["103.86.96.100", "103.86.99.100"]
+DNS_NORD_IPV6 = ["2400:bb40:4444::100", "2400:bb40:8888::100"]
 
-# returns True when Threat Protection Lite DNS servers are used
-def _is_set_to_threat_protection_lite():
-    return "103.86.96.96" and "103.86.99.99" in sh.cat("/etc/resolv.conf")
+# Used for test parametrization.
+DNS_TPL = ["103.86.96.96", "103.86.99.99"]
+DNS_TPL_IPV6 = ["2400:bb40:4444::103", "2400:bb40:8888::103"]
 
+# Used for test parametrization, when the same test has to be run for different values of custom dns.
+DNS_CASE_CUSTOM_SINGLE = "2.0.0.0"
+DNS_CASE_CUSTOM_DOUBLE = "2.0.0.1 2.0.0.2"
+DNS_CASE_CUSTOM_TRIPLE = "2.0.0.3 2.0.0.4 2.0.0.5"
+DNS_CASES_CUSTOM = [DNS_CASE_CUSTOM_SINGLE, DNS_CASE_CUSTOM_DOUBLE, DNS_CASE_CUSTOM_TRIPLE]
 
-# returns True when NordVPN DNS servers are used
-def _is_set_to_nord_ipv6():
-    return "2400:bb40:8888::100" and "2400:bb40:4444::100" in sh.cat("/etc/resolv.conf")
+ALL_DNS_CASES = \
+    DNS_NORD + \
+    DNS_NORD_IPV6 + \
+    DNS_TPL + \
+    DNS_TPL_IPV6 + \
+    DNS_CASE_CUSTOM_SINGLE.split(" ") + \
+    DNS_CASE_CUSTOM_DOUBLE.split(" ") + \
+    DNS_CASE_CUSTOM_TRIPLE.split(" ")
 
 
-# returns True when Threat Protection Lite DNS servers are used
-def _is_set_to_threat_protection_lite_ipv6():
-    return "2400:bb40:4444::103" and "2400:bb40:8888::103" in sh.cat("/etc/resolv.conf")
+def is_unset() -> bool:
+    """ returns True when NordVPN DNS servers are used """
```

Comment:
Comment doesn't seem to be correct. From the code I understand that True will be returned if none of the resolver DNS addresses are the same as the DNS addresses in `ALL_DNS_CASES`. So I think that the comment that was previously here (`returns True when NordVPN app has not modified the DNS`) would be correct.